packages/google_maps_flutter/google_maps_flutter/lib/src/controller.dart

```diff
@@ -98,14 +98,10 @@ class GoogleMapController {
   /// platform side.
   ///
   /// The returned [Future] completes after listeners have been notified.
-  Future<void> _updateMapOptions(Map<String, dynamic> optionsUpdate) async {
+  Future<void> _updateMapOptions(Map<String, dynamic> optionsUpdate) {
```

Comment:
nits: I still think the change from aysnc await to directly returning future is not related to the description?purpose of this PR. It might cause confusion when tracing git history. WDYT? I would remove these and work on them in a seperate PR if really necessary.

Reply:
I can revert these, but all the new plugins which we have federated are using the new 'return the future directly so no need for this to be async' style. I think it'd be much weirder to leave this as async/await than to do the migration at once, but I guess I can have a quick follow up PR with the de-asyncification if you feel strongly about this :)

Reply:
(We discussed this offline, and we agreed to leave these as they are)